README.md

```diff
@@ -1,69 +1,405 @@
 ## Protocol
-- [Protocol specification](https://github.com/aeternity/protocol) - Documentation about the the aeternty protocol
+The specification of the aeternity protocol can be found here:
+- https://github.com/aeternity/protocol
+
+For off-protocol standard proposals check out the AEXpansions:
+- https://github.com/aeternity/AEXs
+
+## Infrastructure & core components
+
+### Node and Middleware
+- [aeternity](https://github.com/aeternity/aeternity) - The official aeternity node written in Erlang.
+- [ae_mdw](https://github.com/aeternity/ae_mdw) - The official aeternity middleware written in Elixir.
+    - Uses a plugin architecture where the node itself serves as extension to the middleware.
+
+**Note**
+- You can download the latest release as well as backups for mainnet or testnet here: https://downloads.aeternity.io
+- For the middleware currently no snapshots exists and a full sync is required
+
+Community developed middleware:
+- [aepp-middleware-mn](https://github.com/kryztoval/aepp-middleware-mn) - Middleware implementation written in Node.js.
+
+### Virtual Machine
+- [aebytecode](https://github.com/aeternity/aebytecode) - Library and stand alone assembler for aeternity bytecode.
+
+## Official Hosted Services
 
-## Node and middleware
-- [Node](https://github.com/aeternity/aeternity) - The official aeternity node written in erlang
-- [Node documentation](http://docs.aeternity.io) - Documentation about operating the official node implementation
-- [Aeternal](https://github.com/aeternity/aeternal) - Node middleware
+### Http compiler
+- https://compiler.aeternity.io/version
 
-## Smart contracts
-- [Sophia](https://github.com/aeternity/protocol/blob/master/contracts/sophia.md) - The Aeternity smart contract language
-- [Sophia CLI](https://github.com/aeternity/aesophia_cli) - The command line client for the Sophia compiler
-- [Sophia HTTP](https://github.com/aeternity/aesophia_http) - The http server exposing the Sophia compiler
+### Node & middleware
+###### Mainnet
+- https://mainnet.aeternity.io/v3/status - The status endpoint of the Node API.
+- https://mainnet.aeternity.io/mdw/status - The status endpoint of the Middleware API.
+
+###### Testnet
+- https://testnet.aeternity.io/v3/status - The status endpoint of the Node API.
+- https://testnet.aeternity.io/mdw/status - The status endpoint of the Middleware API.
+
+**Note**
+- Everybody is encouraged to host their own infrastructure
+    - https://blog.aeternity.com/why-run-an-ae-node-and-how-to-do-it-8b95a685f683
+- You should ***never*** rely on the hosted infrastructure of aeternity for applications running in production!
+
+## Explorers
+
+### Mainnet
+- [AEknow](https://aeknow.org) - Explorer that also shows mining stats developed by the community.
+    - https://github.com/jdgcs/aeknow
+- [aenalytics](https://aenalytics.org) - Explorer with focus on UX and the aeternity naming system (AENS).
+- [Indaex](https://explorer.aeternity.io) - Default explorer provided by aeternity.
```

Comment:
Indaex is not the correct name, right?

Reply:
Good question. There was a branding vote few months ago. I think it should be branded like this then. But agree, currently it isn't called "Indaex". Mhhh 😅

Reply:
https://forum.aeternity.com/t/middleware-new-branding-vote/8186/14

Reply:
But I guess it also wasn't clear if the branding is targeting the backend or the frontend of the middleware. Or even both.

Reply:
Renamed to `aeternity explorer`